android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -398,6 +432,12 @@ public void run() {
                 initializePlayerControl();
                 setControls(controls);
                 applyModifiers();
+
+                //Use Media Session Connector from the ExoPlayer library to enable MediaSession Controls in PIP.
+                MediaSessionCompat mediaSession = new MediaSessionCompat(getContext(), "tag");
```

Comment:
maybe media session management should be its own feature, independent of PIP support?

Reply:
Yeah, could be of course. Should I make another PR for the standalone media session management?

Reply:
https://github.com/react-native-community/react-native-video/pull/1780